server/routes/api/recommendations/recommendationsRoutes.ts

```diff
@@ -58,6 +58,40 @@ recommendationsRouter.get(
     }
   }
 );
+recommendationsRouter.get(
+  "/babysitter/rating/:babysitter",
+  [
+    param("babysitter")
+      .notEmpty()
+      .isNumeric()
+      .withMessage(BABYSITTER_INVALID_INPUT_ERROR),
+  ],
+  async (req: Request, res: Response) => {
+    try {
+      const fieldValidationResult = validationResult(req);
+      if (!fieldValidationResult.isEmpty()) {
+        return res.status(400).json({
+          message: fieldValidationResult
+            .array()
+            .map((item) => item.msg)
+            .join(" "),
+        });
+      }
+      const { babysitter: babysitterId } = req.params;
```

Comment:
What about adding a validation function that verifies that the babysitter Id in the param is known in the db?